src/libs/ReportUtils.js

```diff
@@ -70,7 +71,7 @@ let doesDomainHaveApprovedAccountant;
 Onyx.connect({
     key: ONYXKEYS.ACCOUNT,
     waitForCollectionCallback: true,
-    callback: val => doesDomainHaveApprovedAccountant = val.doesDomainHaveApprovedAccountant,
+    callback: val => doesDomainHaveApprovedAccountant = val.doesDomainHaveApprovedAccountant || false,
```

Comment:
I have been running to this value not being defined locally and causing the app not to load so casting this to false in case `val` is not defined helped. Should be no harm in this change